src/test/java/com/ably/kafka/connect/DefaultChannelSinkMappingTest.java

```diff
@@ -0,0 +1,52 @@
+package com.ably.kafka.connect;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DefaultChannelSinkMappingTest {
+    private DefaultChannelSinkMapping SUT;
```

Comment:
Is the "system under test" also written in uppercase letters in other tests in this codebase? :thinking: If not then maybe we should change it to the usual camel-case style `sut`? :wink:

Reply:
https://github.com/ably/kafka-connect-ably/pull/40/commits/8a736b42ac065a9c409108a444281617c3993e90